.github/workflows/ok-to-test.yml

```diff
@@ -18,7 +18,8 @@ name: Ok To Test
 on:
   issue_comment:
     types: [created]
-
+permissions:
+  pull-requests: write
```

Comment:
my bad: I LGTM'ed the CL but that's not going to pass the scorecard check. The check encourages devs to use `permission:read-all` or `permissions: XXX:read` or `permissions:` globally at the top of the file; and use `write` permission in each job individually. The motivation here is that if we encourage users to set write permission globally, any time they add new actions in the file it will inherit this permission. We d'like to avoid this situation.

Reply:
> my bad: I LGTM'ed the CL but that's not going to pass the scorecard check. The check encourages devs to use `permission:read-all` or `permissions: XXX:read` or `permissions:` globally at the top of the file; and use `write` permission in each job individually. The motivation here is that if we encourage users to set write permission globally, any time they add new actions in the file it will inherit this permission. We d'like to avoid this situation.
> 
> I'm going to start looking at known actions and the permissions they need, to see if we can go further than that and flag write permission when they are not actually needed.
> 
> Advice welcome for this.

The issue with this is a repository could have global reads within the repository setting and explicit write requests for each file and scorecard would fail. Most of the users would turn off this setting as it is more work for them to do this.